authy-react/src/Login.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+
+export default class Login extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            username: '',
+            password: ''
+        }
+    }
+
+    input = (e) => {
```

Comment:
note that the convention and what you will see in most production code is to append Handler to methods that are called on events, i.e. inputHandler and submitHandler